mobile/src/people/avatar.tsx

```diff
@@ -53,7 +53,7 @@ export class Avatar extends Component<AvatarProps, AvatarState> {
         const mimeType = this.validateMimeType(this.props.photo);
         const photoBase64 = this.validateEncodedImage(this.props.photo);
 
-        const image = !mimeType || !photoBase64 ? require('../../src/people/userpic.png') : { uri: mimeType + photoBase64 };
+        const image = !mimeType || !photoBase64 ? require('./userpic.png') : { uri: mimeType + photoBase64 };
 
         const outerFrameFlattenStyle = StyleSheet.flatten([
```

Comment:
I think all styles in this file should be replaced to separate  file.

Reply:
Lines 5-21 is not calculated from props and can be replaced. If PR is not about it and have no permission to replace it - ok.